packages/react-i18n/src/babel-plugin/generate-dictionaries.ts

```diff
@@ -70,5 +71,9 @@ async function readLocaleTranslations(
     filePath.endsWith(`/${locale}.json`),
   );
 
-  return translationPath ? fs.readJson(translationPath) : {};
+  return translationPath
+    ? JSON.parse(
+        stripJsonComments((await fs.readFile(translationPath)).toString()),
```

Comment:
I think if you specify an encoding then readFile returns a string

```
stripJsonComments(await fs.readFile(translationPath, 'utf8')),

Reply:
That's better, thanks 😄